core/src/utils.jl

```diff
@@ -327,3 +327,39 @@ function basin_bottoms(
     bottom_b = something(bottom_b, bottom_a)
     return bottom_a, bottom_b
 end
+
+"""
+Replace the truth states in the logic mapping which contain wildcards with
+all possible explicit truth states.
+"""
+function expand_logic_mapping!(logic_mapping::Dict{Tuple{Int, String}, String})::Nothing
+    keys_old = collect(keys(logic_mapping))
+
+    for (node_id, truth_state) in keys_old
+        pattern = r"^[ $(TF*)* ]+$"
```

Comment:
Doesn't this also allow spaces? Do we want that? Doesn't `^[TF\*]+$` also capture the options?